strr-platform-web/app/components/modal/Base.vue

```diff
@@ -0,0 +1,85 @@
+<script setup lang="ts">
+const modalModel = defineModel({ type: Boolean, default: false })
+const isSmallScreen = useMediaQuery('(max-width: 640px)')
```

Comment:
Oh nice, I think I'm going to update the fee widget to use this instead of the 'isFoldable'